pysatNASA/instruments/icon_mighti.py

```diff
@@ -234,7 +234,7 @@ def clean(self):
 
 # Set the download routine
 dirstr1 = '/pub/data/icon/l2/l2-1_mighti-{{id:s}}_los-wind-{color:s}/'
-dirstr2 = '/pub/data/icon/l2/l2-2_mighti-vector-wind-{color:s}/'
```

Comment:
Is it bad that I want to suggest replacing the `_` with `?` 😝

Reply:
Nope. Since no `-` or `_` is safe the whole thing would be `?`. 🤷